gateway/handler_codec.go

```diff
@@ -182,7 +182,7 @@ func (i *handler) serveCodecRaw(ctx context.Context, w http.ResponseWriter, r *h
 
 // serveCodecConverted returns payload converted to codec specified in toCodec
 func (i *handler) serveCodecConverted(ctx context.Context, w http.ResponseWriter, r *http.Request, resolvedPath ipath.Resolved, contentPath ipath.Path, toCodec mc.Code, modtime time.Time) {
-	obj, err := i.api.Dag().Get(ctx, resolvedPath.Cid())
+	obj, err := i.api.GetNode(ctx, resolvedPath.Cid())
```

Comment:
:thought_balloon:  This seems to be the only place we use `Get[IPLD]Node`.

Reply:
Yeah, I don't think we can get "rid of" IPLD since we have the conversions.

Reply:
Actually managed to remove it :D